pkg/provider/constants.go

```diff
@@ -71,15 +71,9 @@ const gcpBackendTemplate = `terraform {
   }
 }
 
-locals {
-  gcp_location  = {{ .Values.Location | quote }}
-  gcp_location_parts = split("-", local.gcp_location)
-  gcp_region         = "${local.gcp_location_parts[0]}-${local.gcp_location_parts[1]}"
-}
-
 provider "google" {
   project = {{ .Values.Project | quote }}
-  region  = local.gcp_region
+  region  = {{ .Values.Region | quote }}
```

Comment:
this should no longer be used I thought?

Reply:
These templates are indeed no longer used.

Reply:
We should remove then